Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/RedLeftBlueRightAuto.java

```diff
@@ -1,42 +1,43 @@
 package org.firstinspires.ftc.team417_PowerPlay;
 
 import com.acmerobotics.roadrunner.geometry.Pose2d;
-import com.acmerobotics.roadrunner.geometry.Vector2d;
 import com.acmerobotics.roadrunner.trajectory.Trajectory;
 import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
 import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
 
 import org.firstinspires.ftc.team417_PowerPlay.drive.SampleMecanumDrive;
 
-@Autonomous (name="strafe test")
-public class TestAutonomous extends LinearOpMode {
+@Autonomous (name="Red left || BLue right")
+public class RedLeftBlueRightAuto extends LinearOpMode {
 
     // Coordinates for various tiles referenced on page 46 of:
     // https://www.firstinspires.org/sites/default/files/uploads/resource_library/ftc/game-manual-part-2-traditional.pdf
     // Robot's position starts at (0, 0) - all tile coordinates are in relation to this starting position
-    private static final double Y_POSITION_A = -30;
-    private static final double X_POSITION_1 = 41;
-    private static final double X_POSITION_AWAY_FROM_WALL = 5;
+    private static final double Y_POS_AWAY_FROM_WAll = 8;
+    private static final double X_POS_1 = 33;
+    private static final double Y_POS_E = 20;
```

Comment:
If you're aiming for a specific tile, shouldn't it be like A5 like the game manual says? Not just 1 and E separately.

Reply:
Idk if this makes the most sense, maybe there's another way to have the constants but we just did the rows and columns as X and Y positions respectively because the trajectory builder requires X and Y values as separate ints. But I added a comment to better explain the constants

Reply:
Oh so X is row 1 and Y is column E? I guess that makes sense. Maybe add a comment saying that?